dm/config/subtask.go

```diff
@@ -189,6 +189,8 @@ type SubTaskConfig struct {
 
 	CleanDumpFile bool `toml:"clean-dump-file" json:"clean-dump-file"`
 
+	EnableANSIQuotes bool `toml:"ansi-quotes" json:"ansi-quotes"`
```

Comment:
rename to `ansi-quotes` to avoid conflict with deprecated sycncer config `enable-ansi-quotes`